tezos_interop/tezos_interop.re

```diff
@@ -808,6 +809,42 @@ module Consensus = {
       ~on_message,
     );
   };
+  let fetch_validators = (~context) => {
+    let Context.{rpc_node, required_confirmations, consensus_contract, _} = context;
+    let micheline_to_validators =
+      fun
+      | Ok(
+          Micheline.Prim(
+            _,
+            Michelson_v1_primitives.D_Pair,
+            [Prim(_, D_Pair, [_, Seq(_, keys)], _), _, _],
+            _,
+          ),
+        ) => {
+          fold_left_ok(
+            (acc, k) =>
+              switch (k) {
+              | Micheline.String(_, k) =>
+                switch (Key.of_string(k)) {
+                | Some(k) => Ok([k, ...acc])
```

Comment:
This reverts the order of validators, and it shouldn't, the order of validators is quite important for the consensus, as it's how we decide on the next validator.

Reply:
Added a List.rev